frontend/src/common/components/OwnershipList.tsx

```diff
@@ -34,38 +34,40 @@ const OwnershipList = () => {
                         </legend>
                     </li>
 
-                    {fields.map((field, index) => (
-                        <li
-                            className="ownership-item"
-                            key={field.id}
-                        >
-                            <div className="owner">
-                                <RelatedModelInput
-                                    label="Omistaja"
-                                    required
-                                    queryFunction={useGetOwnersQuery}
-                                    relatedModelSearchField="name"
-                                    name={`ownerships.${index}.owner`}
-                                    transform={(obj) => formatOwner(obj)}
-                                    RelatedModelMutateComponent={OwnerMutateForm}
-                                />
-                            </div>
-                            <div className="percentage">
-                                <NumberInput
-                                    name={`ownerships.${index}.percentage`}
-                                    allowDecimals
-                                    required
-                                />
-                                <span>%</span>
-                            </div>
-                            <div className="icon--remove">
-                                <IconCrossCircle
-                                    size="m"
-                                    onClick={() => remove(index)}
-                                />
-                            </div>
-                        </li>
-                    ))}
+                    {fields.map((field, index) => {
+                        return (
```

Comment:
return added again by your IDE probably Anni :D

Reply:
fixed again, not an IDE issue, the branch was based on the other branch before I fixed it